packages/vector_graphics_compiler/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.1.16
+
+* Fixes an issue when parsing stroke-width with an invalid value
```

Comment:
This is missing a period; please see the CHANGELOG style guide linked from the checklist.